tests/Model/DataType/LocalizedFieldTest.php

```diff
@@ -92,4 +93,48 @@ public function testLocalizedFieldInsideFieldCollection()
         //new value (de): index 0
         $this->assertEquals('textDE', $loadedItem->getLinput('de'), 'New localized value inside fieldcollection not saved or loaded properly');
     }
+
+    public function testLocalizedFieldFallback()
+    {
+        $configuration = Config::getSystemConfiguration();
+        $configuration['general']['fallback_languages']['de'] = 'en';
+        Config::setSystemConfiguration($configuration);
+
+        $object = TestHelper::createEmptyObject();
+
+        //en values
+        $object->setLinput('TestEN', 'en');
+        $object->setLcheckbox(true, 'en');
+        $object->setLnumber(123, 'en');
+
+        //de values
+        $object->setLinput('TestDE', 'de');
+        $object->setLcheckbox(true, 'de');
+        $object->setLnumber(0, 'de');
+
+        $object->save();
+
+        //empty de values check fallback
+        $object->setLinput('', 'de');
+        $object->setLcheckbox(false, 'de');
```

Comment:
can't we set `null` here ... as it says we empty the values?

Reply:
we also need to set number to null and check if fallback works properly

Reply:
I used the value possible to set from Admin UI

Reply:
checking both cases now, when setting `null` & `false`(special case)